docs/source/changes.rst

```diff
@@ -1,20 +1,48 @@
 Changelog
 =========
 
-v2.0 (Development)
-------------------
+v2.0 (November 1, 2018)
+-----------------------
 
-- Python 2 is no longer supported
+PyQuil 2.0 is a major release of pyQuil: Rigetti's toolkit for constructing and running quantum
+programs. This release contains many major changes reflecting:
+
+1. The introduction of `Quantum Cloud Services <https://www.rigetti.com/qcs>`_. Access
+   Rigetti's QPUs from co-located classical compute resources for minimal latency. The
+   web API for running QVM and QPU jobs has been deprecated and cannot be accessed with
+   pyQuil 2.0
+2. Advances in classical control systems and compilation allowing the pre-compilation of
+   parametric binary executables for rapid hybrid algorithm iteration.
+3. Changes to Quil---our quantum instruction language---to provide easier ways of
+   interacting with classical memory.
+
+The new QCS access model and features will allow you to execute hybrid quantum algorithms
+several orders of magnitude (!) faster than the previous web endpoint. However, to fully
+exploit these speed increases you must update your programs to use the latest pyQuil features
+and APIs. Please read :ref:`quickstart` for a comprehensive migration guide.
```

Comment:
this renders as ":ref:`quickstart`" when I look at the page source in the github browser

Reply:
it also doesnt seem to link correctly

Reply:
for better or worse, the tag for `migration.rst` is called `quickstart`

Reply:
I'm saying that it doesnt link or render correctly, not that the RST tag is wrong

Reply:
it renders correctly if you render it. It won't work from github